NDTensors/src/abstractarray/ndims.jl

```diff
@@ -1,10 +1,26 @@
 ## NDTensors.ndims (not imported from Base)
+using .TypeParameterAccessors: set_parameter
 
-ndims(array::AbstractArray) = Base.ndims(array)
-ndims(arraytype::Type{<:AbstractArray}) = Base.ndims(arraytype)
+ndims((array::AbstractArray)) = ndims(typeof(array))
+ndims(arraytype::Type{<:AbstractArray}) = parameter(arraytype, Base.ndims)
```

Comment:
Note that `TypeParameterAccessors.parameter` is supposed to be an internal function to `TypeParameterAccessors`, you should use `type_parameter` instead. `type_parameter` errors if a parameter is unspecified, which is the behavior we want for functions like `ndims`.

Reply:
I updated this, thanks!